nss-day-cohort-60/national-parks-django-api
# Jd event registration feature

### Changes
Added event registration inside event post method

###Testing

1. Pull JD-event-registration-feature branch and switch to it
2. run server
3. in client as non-staff user, register for an event by clicking on an existing event and clicking register button
4. check database for new registration for that event
